src/frontend/apps/standalone_site/src/components/Cards/ContentCard/ContentCard.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Box,
+  Text,
+  Card as GrommetCard,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  BoxExtendedProps,
+  BoxTypes,
+} from 'grommet';
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const CardBox = styled(Box)`
+  gap: 1.7rem;
+`;
+export const ContentCards = ({
+  children,
+  ...boxProps
+}: PropsWithChildren<BoxExtendedProps>) => {
+  return (
+    <CardBox direction="row" wrap={true} justify="start" {...boxProps}>
+      {children}
+    </CardBox>
+  );
+};
+
+interface CardLayoutPropsExtended {
+  opacity: number;
+}
+const CardLayout = styled(GrommetCard)<CardLayoutPropsExtended>`
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.2);
+  opacity: ${(props) => props.opacity};
+  transition: opacity 0.6s;
+  cursor: pointer;
+`;
+
+interface ContentCardProps extends BoxTypes {
+  header: React.ReactNode;
+  footer?: React.ReactNode;
+  title: string;
+}
+
+export const ContentCard = ({
+  header,
+  footer,
+  title,
+  children,
+  ...cardLayoutProps
+}: PropsWithChildren<ContentCardProps>) => {
+  const [opacity, setOpacity] = useState(0);
+  useEffect(() => {
+    setOpacity(1);
+  }, []);
+
+  return (
+    <CardLayout
+      height="inherit"
+      width="234px"
```

Comment:
What do you suggest?
width="small" => 192px  -  It is a bit too small
width="medium" =>  384px;  -  It is a too big
Or in REM? It is about 15rem.

Let me know

Reply:
And what about a width in % with a min and a max set ?

Reply:
If I put % the width will move depend if the menu is open or not, you will loose the possibility to get more cards by row when the menu close.
If I want to use min and max it will be something like that `width={{ min: '234px', max: '234px' }}`, I find that a bit strange personally, otherwise each card will have his own width.

![Screenshot (116)](https://user-images.githubusercontent.com/25994652/196155812-222d711a-eb7f-43b9-9be1-2ec3099d0348.png)
![Screenshot (117)](https://user-images.githubusercontent.com/25994652/196155820-e3ee7e3d-b561-4f29-9a95-50831f9088e8.png)